client-side/src/i18n.js

```diff
@@ -0,0 +1,15 @@
+import i18n from 'i18next'
+import { initReactI18next } from "react-i18next";
+import languageDetector from 'i18next-browser-languagedetector';
+import Backend from 'i18next-http-backend'
+i18n
+.use(initReactI18next)
+.use(languageDetector)
+.use(Backend)
+.init(
+    {
+        debug:true,
+        fallbackLng:'he'
```

Comment:
fallbackLng ahould be same as default language (that you declared as en), please change